libs/wcm/constants/permissions.js

```diff
@@ -1,5 +1,5 @@
 export const SIGNING_METHODS = {
```

Comment:
NRTTF (not related to this file)
I dont see any code in this PR that requires us to update our package-lock.json files

Reply:
I don't understand this comment

Reply:
Just wondering why the lock files are pushed for this PR? What new packaged do we need to update to make the changes in this PR run?